sdk/Lusid.Sdk.Tests/tutorials/Instruments/EquitySwapExamples.cs

```diff
@@ -164,10 +194,101 @@ public void EquitySwapValuationExampleWithExposureAndAccruedInterest(ModelSelect
             var exposure = (double) data[exposureKey];
             Assert.That(exposure, Is.GreaterThanOrEqualTo(0));
             
-            // CHECK accrued interest
-            // TODO: Looks like accrued interest is null, not clear if this is correct.
-            var accruedInterest = data[accruedInterestKey];
+            // CHECK accrued interest is returned and is not zero (for equity swaps, it can be positive or negative).
+            var accruedInterest = (double) data[accruedInterestKey];
+            Assert.That(accruedInterest, Is.Not.EqualTo(0).Within(1e-3));
+            
+            // CLEAN up.
+            _recipeApi.DeleteConfigurationRecipe(scope, recipeCode);
+            _instrumentsApi.DeleteInstrument("ClientInternal", instrumentID);
+            _portfoliosApi.DeletePortfolio(scope, portfolioCode);
+        }
+        
+        [LusidFeature("F22-44")] 
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney)]
+        [TestCase(ModelSelection.ModelEnum.Discounting)]
+        public void LifeCycleManagementForEquitySwap(ModelSelection.ModelEnum model)
+        {
+            // CREATE an EquitySwap
+            var equitySwap = InstrumentExamples.CreateExampleEquitySwap();
+            
+            // CREATE wide enough window to pick up all cashflows associated to the EquityOption
```

Comment:
In these comments you're mentioning an EquityOption. Should this be equity swap?